modules/programs/git-workspace.nix

```
@@ -0,0 +1,10 @@
+{ config, lib, pkgs, ... }:
+let cfg = config.programs.git-workspace;
+in {
+  meta.maintainers = [ lib.maintainers.aciceri ];
+  options.programs.git-workspace = {
+    enable = lib.mkEnableOption "git-workspace";
+    package = lib.mkPackageOption pkgs "git-workspace" { };
+  };
+  config = lib.mkIf cfg.enable { home.packages = [ pkgs.git-workspace ]; };
```

I assume you meant to use

```suggestion
  config = lib.mkIf cfg.enable { home.packages = [ cfg.package ]; };
```

here.

But I would suggest simply adding

    home.packages = [ cfg.package ];

to the service module and removing this file entirely.

---

Yes, having a `progams` module like this is pretty insignificant indeed, I'm removing it.